src/main/resources/pipeline/circleci/config.yml.mustache

```diff
@@ -0,0 +1,51 @@
+version: 2.1
+
+jobs:
+  build:
+    docker:
+      - image: cimg/openjdk:8.0.322
```

Comment:
the default project version use java 11, what do you think if you upgrade this image to use java 11?.

Reply:
It is true, the version is updated to java 11 but I think that in the future adding according to the version of the project is the best, I hope to contribute.